### PR TITLE
chore: simplify ESM `execute` example

### PR DESCRIPTION
**Pull request by mon-jai**

### CODE REVIEW

**Comment by mon-jai** [comment]
Signed

File: src/execute.ts [hunk 1]
```diff
@@ -13,23 +13,17 @@ import {settings} from './settings'
...
  * ```
  * #!/usr/bin/env -S node --loader ts-node/esm --no-warnings=ExperimentalWarning
- * async function main() {
- *   const oclif = await import('@oclif/core')
- *   await oclif.execute({development: true, dir: import.meta.url})
- * }
+ * import { execute } from "@oclif/core"
  *
- * await main()
+ * await oclif.execute({development: true, dir: import.meta.url})
```
Comment (mdonnalley) [inline src/execute.ts:18]: With `import {execute} from '@oclif/core'`, there's no longer an oclif namespace - so it should now be `execute({development: true, dir: import.meta.url})`

**Review by mdonnalley — CHANGES_REQUESTED** [review]

**Review by mdonnalley — APPROVED** [review]

**Comment by mdonnalley** [comment]
Thanks @mon-jai 🏆

**Comment by mon-jai** [comment]
@mdonnalley Follow-up PR for [the ESM template](https://github.com/oclif/hello-world-esm/pull/129) and [documentations](https://github.com/oclif/oclif.github.io/pull/223).

**Pull request merged**